client/src/pages/game/GamePrompt.js

```diff
@@ -1,10 +1,28 @@
-import React from "react";
+import React, { useState, useEffect, useContext } from "react";
 import { Box, Container, Grid, Typography } from "@material-ui/core";
 import TimerIcon from "@material-ui/icons/Timer";
-
+import { AppContext } from "../../App";
 import { TEAM_ROLE } from "../game_lobby/team_select/TeamPresets";
 
 function GamePrompt({ gameState, player }) {
+  const { gameIO } = useContext(AppContext);
+
+  const [timer, setTimer] = useState(45);
+
+  let countDown = null;
+
+  gameIO.state.io.on("start timer", () => {
+    clearTimeout(countDown);
+    setTimer(45)
```

Comment:
We should use and add the server's countdown time to gameState.gameTurn (maybe as gameState.gameTurn.timeLeft).

The reason for this is, if the user refreshes the page during the game, I believe the countdown time would be incorrect. By using the current countdown time from the gameState when the game page reloads, we can guarantee it to be accurate.